apps/web/vibes/soul/primitives/accordions/index.tsx

```diff
@@ -40,12 +44,22 @@ function Accordion({
   }, []);
 
   return (
-    <AccordionsPrimitive.Item {...rest}>
+    <AccordionsPrimitive.Item
+      {...props}
+      className={clsx(
+        'focus:outline-2 has-[:focus-visible]:ring-2 has-[:focus-visible]:ring-[var(--accordion-focus,hsl(var(--primary)))] has-[:focus-visible]:ring-offset-4',
+        {
+          light: 'ring-offset-[var(--acordion-light-offset,hsl(var(--background)))]',
+          dark: 'ring-offset-[var(--acordion-dark-offset,hsl(var(--foreground)))]',
+        }[colorScheme],
+        className,
```

Comment:
Have we considered that if we use `clsx` like this to concatenate classes we probably also need `tailwind-merge`? I know @apledger's not a huge fan of it but if we're going down the route of allow to override classes, we should probably use it